Cargo.lock

```diff
@@ -655,13 +665,26 @@ dependencies = [
  "serde",
 ]
 
+[[package]]
+name = "dprint-plugin-markdown"
+version = "0.4.3"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "f2d00c846eeffb9c741373b67c7b8aecc1b191829856f61b63bfc94da7f2065c"
+dependencies = [
+ "dprint-core 0.31.1",
+ "lazy_static",
+ "pulldown-cmark",
+ "regex",
+ "serde",
+]
+
 [[package]]
 name = "dprint-plugin-typescript"
 version = "0.38.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "67bb41b131e30df80105eb6b4f183a4bf003ee7dfb20661ac15a2a6f329de5b1"
 dependencies = [
- "dprint-core",
+ "dprint-core 0.34.0",
```

Comment:
This is unfortunate, but not a blocker.

Reply:
I think we could upgrade `dprint` before landing to ensure only a single version

Reply:
Opened https://github.com/dprint/dprint-plugin-markdown/pull/29

Reply:
Resolved in markdown 0.4.4, typescript 0.38.2

Reply:
Thanks!